ui/src/app/features/user-list/user-list.component.ts

```diff
@@ -67,6 +72,8 @@ export class UserListComponent implements OnInit, OnDestroy {
     this.availableRolesSubscription = this.userListFacade.availableRoles$.subscribe(value => this.availableRoles = value);
     this.currentUserId$ = this.userListFacade.currentUserId$;
     this.currentUserEmail$ = this.userListFacade.currentUserEmail$;
+    this.translate.get("org_users.add_users_title").subscribe(title => this.messageHeader = title);
```

Comment:
memory leak. need to unsubscribe. better to use translate.instant(). Given the work in another PR, this will be redundant at all.